mats/misc.ms

```diff
@@ -1310,9 +1310,16 @@
   (equal? (getenv "FUBULY") "FRATZ")
   (eq? (putenv "FUBULY" "fratz") (void))
   (equal? (getenv "FUBULY") "fratz")
+  (eq? (putenv "FUBULY" "") (void))
+  (equal? (getenv "FUBULY") "")
   (error? (getenv 'hello))
   (error? (putenv 'hello "goodbye"))
   (error? (putenv "hello" 'goodbye))
+  (error? (putenv '() "goodbye"))
+  (error? (putenv "hello" '()))
```

Comment:
Chez Scheme requires the arguments to putenv to be strings, so the two cases with () aren't necessary.

Reply:
Done